plugins/wpe-headless/tests/install-wp-tests.sh

```diff
@@ -0,0 +1,155 @@
+r#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
Thanks! Non-interrupting typo for the win.